src/main.rs

```diff
@@ -89,21 +88,15 @@ fn main() {
         None
     };
 
-    let tpu_client_pool = Arc::new(RotatingQueue::<Arc<TpuClient<QuicPool, QuicConnectionManager, QuicConfig>>>::new(
-        number_of_tpu_clients,
-        || {
-            let quic_connection_cache = quic_connection_cache.clone();
-            Arc::new(
-                TpuClient::new_with_connection_cache(
-                    rpc_clients.get().clone(),
-                    &websocket_url,
-                    solana_client::tpu_client::TpuClientConfig::default(),
-                    quic_connection_cache.unwrap(),
-                )
-                .unwrap(),
-            )
-        },
-    ));
+    let tpu_client = Arc::new(
+        TpuClient::new_with_connection_cache(
+            rpc_client.clone(),
+            &websocket_url,
+            solana_client::tpu_client::TpuClientConfig::default(),
```

Comment:
As all the other types are imported, why not also import this one?

```suggestion
   use {
       ...
       solana_client::tpu_client::TpuClientConfig,
       ...
   };
   
            TpuClientConfig::default(),
```